### DISCUSSION TITLE
Page build failure and no update on website

### DISCUSSION THREAD

**Discussion by chris-scientist** [community/community#23963] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I have an website on GitHub Pages (<a href="https://github.com/chris-scientist/gamebuino" class="inline-onebox" rel="noopener nofollow ugc">GitHub - chris-scientist/gamebuino: API reference (EN only) and Academy (FR and EN)</a>). I have recently modified a page, but my website has not been updated. And I received an email (Page build failure), do you know how to correct the problem?</p>
<p>Thank you in advance</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/chris-scientist">@chris-scientist</a>, welcome to the GitHub Support Community! It looks like there was <a href="https://www.githubstatus.com/incidents/wdhkj70kynwk">an incident</a> related to GitHub Pages availability on the day your site build failed. This is likely why you saw an error when building your site.</p>
<p>It looks like a push 2 days ago built successfully though, so the latest version of your site should be live!</p>

**Comment by chris-scientist** [comment]
<p>Hi,</p>
<p>Yes, the last build is success !</p>
<p>Thanks</p>
